### PR TITLE
Justin krs/customer nav

### PR DESCRIPTION
**Pull request by justinengle-krs**
The PR includes the Navigation functionality which exists on the Customer Page.

- Create a nav component to render out the tabs
- Create all components for each tab route
- Create story

### CODE REVIEW

File: src/components/Customer/Customer.svelte [hunk 1]
```diff
@@ -4,25 +4,45 @@
...
+    { name: 'Segments', component: Segments },
+    { name: 'Data Sync', component: DataSync },
+    { name: 'Dynamic Pricing', component: DynamicPricing }
+  ];
+
   async function loadData() {
     loading = true;
     try {
       customer = await getCustomerById(customerId);
+      selectedComponent = tabs[0].component;
```
Comment (matthampton-stock2shop) [inline src/components/Customer/Customer.svelte:37]: Is the selected tab not something that we'd want to include in the url route? (feels to me like that would be useful)... Or is it there already?

File: src/components/Customer/Tabs.svelte [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
```
Comment (matthampton-stock2shop) [inline src/components/Customer/Tabs.svelte:1]: Could we maybe add a story for the Tabs component?

**Review by matthampton-stock2shop — COMMENTED** [review]
I'm going to merge this in the #65 and then link to these comments from there

File: src/components/Customer/Tabs.svelte (hunk 2 above)
Comment (justinengle-krs) [reply src/components/Customer/Tabs.svelte:1]: There is one?

CustomerTabs.stories.svelte?

File: src/components/Customer/Customer.svelte (hunk 1 above)
Comment (justinengle-krs) [reply src/components/Customer/Customer.svelte:37]: Let me see what can be done, it's currently just tabs on the` customer/{id}` and defaults to details.

But it should be possible for something like `customer/{id}/orders` or `customer/{id}?tab=Orders`

File: src/components/Customer/Tabs.svelte (hunk 2 above)
Comment (matthampton-stock2shop) [reply src/components/Customer/Tabs.svelte:1]: Just assuming we plan on re-using it - so having a place with 'generic' parameters might be nice - but I guess not necessary - don't worry

**Pull request merged**